android-components/components/browser/state/src/main/java/mozilla/components/browser/state/action/BrowserAction.kt

```diff
@@ -73,10 +73,24 @@ object InitAction : BrowserAction()
 object RestoreCompleteAction : BrowserAction()
 
 /**
- * [BrowserAction] implementation for updating state related to whether the extensions process
- * spawning has been disabled and a popup is necessary.
+ * [BrowserAction] implementations to react to extensions process events.
  */
-data class ExtensionProcessDisabledPopupAction(val showPopup: Boolean) : BrowserAction()
+sealed class ExtensionsProcessAction : BrowserAction() {
+    /**
+     * [BrowserAction] to indicate when the crash popup should be displayed to the user.
+     */
+    data class ShowPopupAction(val showPopup: Boolean) : ExtensionsProcessAction()
```

Comment:
nit: Maybe we could use `ShowPromptAction` to be more consistent with our already existing similar APIs :)

Reply:
Ok, saw you were fallowing `showExtensionProcessDisabledPopup`.